client/sites/staging-site/components/staging-site-sync-modal/index.tsx

```diff
@@ -463,6 +489,19 @@ export default function SyncModal( {
 								/>
 							</VStack>
 						) }
+						<HStack alignment="left" spacing={ 1 }>
+							<Text color="var(--studio-gray-40)">
+								{ displayBackupDate
+									? createInterpolateElement( __( 'Backup contents from: <date />' ), {
```

Comment:
This phrasing sounds a bit confusing to me, it reads like we'd like the user to backup the changes between the displayed date and now:

<img width="1336" height="1020" alt="Image" src="https://github.com/user-attachments/assets/dd980ddd-6c43-4730-a4e3-8623ae960dc6" />

WDYT about using , "Contents of backup at", "Showing contents of backup at", or simply "Latest backup" instead?